core/src/main/scala/cats/data/EitherK.scala

```diff
@@ -234,4 +234,3 @@ private[data] trait EitherKComonad[F[_], G[_]] extends Comonad[EitherK[F, G, ?]]
   def extract[A](p: EitherK[F, G, A]): A =
     p.extract
 }
-
```

Comment:
This is by accident?

Reply:
That's Emacs doing some extra housekeeping from having the file open. I'll revert it.

Reply:
Also-- you're fast! This PR was up for just a few minutes and you were already on top of it 😄.

Reply:
Haha, yeah, sorry for nitpicking so quickly 😄